src/components/BugzillaComponents/index.jsx

```diff
@@ -104,6 +104,8 @@ const getTableHeaders = (data, onComponentDetails) => {
     },
   };
 
+  const getColor = (value, label) => (label === 'P1s defect' && value.count > 0 ? 'red' : 'blue');
```

Comment:
Would it be possible to use the field name P1Defect instead of the label ? 
I would prefer we don't use the label as it might change